--- portmidi/pm_test/latency.c.orig	2009-09-16 18:54:04.000000000 +0200
+++ portmidi/pm_test/latency.c	2013-11-08 22:37:43.746372124 +0100
@@ -280,7 +280,7 @@
 {
     char line[STRING_MAX];
     int n = 0, i;
-    printf(prompt);
+    fputs(prompt, stdout);
     while (n != 1) {
         n = scanf("%d", &i);
         fgets(line, STRING_MAX, stdin);
--- portmidi/pm_test/midiclock.c.orig	2009-10-04 21:50:44.000000000 +0200
+++ portmidi/pm_test/midiclock.c	2013-11-08 22:43:24.679690984 +0100
@@ -167,7 +167,7 @@
 {
     char line[STRING_MAX];
     int n = 0, i;
-    printf(prompt);
+    fputs(prompt, stdout);
     while (n != 1) {
         n = scanf("%d", &i);
         fgets(line, STRING_MAX, stdin);
@@ -256,7 +256,7 @@
     err = Pm_OpenOutput(&midi, outp, DRIVER_INFO, OUTPUT_BUFFER_SIZE, 
                         TIME_PROC, TIME_INFO, LATENCY);
     if (err) {
-        printf(Pm_GetErrorText(err));
+        fputs(Pm_GetErrorText(err), stdout);
         goto error_exit_no_device;
     }
     active = true;
--- portmidi/pm_test/mm.c.orig	2010-10-05 20:49:09.000000000 +0200
+++ portmidi/pm_test/mm.c	2013-11-08 22:47:25.009680022 +0100
@@ -119,7 +119,7 @@
 {
     char line[STRING_MAX];
     int n = 0, i;
-    printf(prompt);
+    fputs(prompt, stdout);
     while (n != 1) {
         n = scanf("%d", &i);
         fgets(line, STRING_MAX, stdin);
@@ -136,7 +136,7 @@
     if (!active) return;
     while ((count = Pm_Read(midi_in, &event, 1))) {
         if (count == 1) output(event.message);
-        else            printf(Pm_GetErrorText(count));
+        else            fputs(Pm_GetErrorText(count), stdout);
     }
 }
 
@@ -168,7 +168,7 @@
     inp = get_number("Type input device number: ");
     err = Pm_OpenInput(&midi_in, inp, NULL, 512, NULL, NULL);
     if (err) {
-        printf(Pm_GetErrorText(err));
+        fputs(Pm_GetErrorText(err), stdout);
         Pt_Stop();
         mmexit(1);
     }
@@ -484,7 +484,7 @@
         "gs", "a", "bf", "b"    };
     /* note octave correction below */
     sprintf(result, "%s%d", ptos[p % 12], (p / 12) - 1);
-    printf(result);
+    fputs(result, stdout);
     return strlen(result);
 }
 
--- portmidi/pm_test/sysex.c.orig	2010-09-20 21:57:48.000000000 +0200
+++ portmidi/pm_test/sysex.c	2013-11-08 22:48:00.546346082 +0100
@@ -39,7 +39,7 @@
 {
     char line[STRING_MAX];
     int n = 0, i;
-    printf(prompt);
+    fputs(prompt, stdout);
     while (n != 1) {
         n = scanf("%d", &i);
         fgets(line, STRING_MAX, stdin);
--- portmidi/pm_test/test.c.orig	2009-09-16 18:54:04.000000000 +0200
+++ portmidi/pm_test/test.c	2013-11-08 22:48:28.869678279 +0100
@@ -37,7 +37,7 @@
 {
     char line[STRING_MAX];
     int n = 0, i;
-    printf(prompt);
+    fputs(prompt, stdout);
     while (n != 1) {
         n = scanf("%d", &i);
         fgets(line, STRING_MAX, stdin);
